control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -2708,8 +2708,19 @@ func (r *HostedControlPlaneReconciler) reconcileOpenShiftControllerManager(ctx c
 }
 
 func (r *HostedControlPlaneReconciler) reconcileOpenShiftRouteControllerManager(ctx context.Context, hcp *hyperv1.HostedControlPlane, observedConfig *globalconfig.ObservedConfig, releaseImageProvider *imageprovider.ReleaseImageProvider, createOrUpdate upsert.CreateOrUpdateFN) error {
+	ocpVersion, err := semver.Parse(releaseImageProvider.Version())
```

Comment:
Don't need to know the version. Seems you are PR against main, this is implicitly >= 4/14
https://github.com/openshift/hypershift/pull/2674/files#r1233683308

Reply:
ack, removed